docs/guides/actions.rst

```diff
@@ -28,6 +28,28 @@ The ``mod/example/actions/example.php`` script will now be run whenever a form i
 
 .. warning:: A stumbling point for many new developers is the URL for actions. The URL always uses ``/action/`` (singular) and never ``/actions/`` (plural). However, action script files are usually saved under the directory ``/actions/`` (plural) and always have an extension.
 
+Registering actions using plugin config file
+--------------------------------------------
+You can also register actions via the :doc:`elgg-plugin</guides/plugins>` config file. 
+To do this you need to provide an action section in the config file. 
+The location of the action files are assumed to be in the plugin folder  ``/actions``.
+
+.. code:: php
+
+	<?php
+
+	return [
+		'actions' => [
+			'logged_in' => [
+				'discussion/save',
```

Comment:
Seem equally readable to me. I like PR as we can easily move from numeric keys to `name => classname`.